internal/guest/runtime/runc/runc.go

```diff
@@ -251,30 +252,31 @@ func (c *container) GetState() (*runtime.ContainerState, error) {
 // It should be noted that containers that have stopped but have not been
 // deleted are still considered to exist.
 func (c *container) Exists() (bool, error) {
-	states, err := c.r.ListContainerStates()
+	// use global path because container may not exist
+	logPath := c.r.getGlobalLogPath()
```

Comment:
Is there a possibility of race condition here? (even before this change). What happens if there are simultaneous requests that all try to write to the same log file?

Reply:
It looks like runc opens the file with `O_SYNC`, so multiple writes will succeed and won't overwrite each other. But there may still be a race between when a runc command finishes and gcs reads the log file (another command may finish in that span and write its own log to the file). I don't know why we use the log files, but I'll explore getting the errors to print on `stderr` and not using the log file in a separate PR.

Reply:
Sounds good, no need to modify that behavior here. I was just curious.